src/pages/leaderboard/[address].tsx

```diff
@@ -0,0 +1,60 @@
+import LeaderboardPage from '@/modules/LeaderboardPage'
+import { getRewardHistoryQuery } from '@/services/datahub/content-staking/query'
+import {
+  getGeneralStatisticsQuery,
+  getLeaderboardDataQuery,
+  getUserStatisticsQuery,
+} from '@/services/datahub/leaderboard/query'
+import { getCommonStaticProps } from '@/utils/page'
+import { QueryClient, dehydrate } from '@tanstack/react-query'
+import { AppCommonProps } from '../_app'
+
+export const getStaticPaths = async () => {
+  // Skip pre-rendering, because it will cause slow build time
+  return {
+    paths: [],
+    fallback: 'blocking',
+  }
+}
+
+export const getStaticProps = getCommonStaticProps<
```

Comment:
@teodorus-nathaniel can you check it, is it make sense use StaticProps for dynamic URLs?

or better use just server render?